### PR TITLE
Increase test coverage in pyscal/utils

### PR DESCRIPTION
**Pull request by berland**

### CODE REVIEW

File: pyscal/utils/relperm.py [hunk 1]
```diff
@@ -11,34 +11,46 @@
...
 
-def crosspoint(dframe: pd.DataFrame, satcol: str, kr1col: str, kr2col: str) -> float:
+def crosspoint(
+    inputframe: pd.DataFrame, satcol: str, kr1col: str, kr2col: str
+) -> float:
     """Locate the crosspoint where kr1col == kr2col
 
     Args:
-        dframe: Dataframe with at least three columns
+        inputframe: Dataframe with at least three columns
```
Comment (asnyv) [inline pyscal/utils/relperm.py:20]: Is it worth breaking api for a change from `dframe` to `inputframe`? At least it doesn't feel natural to do it in a PR on increased test coverage?

**Review by asnyv — COMMENTED** [review]
one comment

File: pyscal/utils/relperm.py (hunk 1 above)
Comment (berland) [reply pyscal/utils/relperm.py:20]: Its not worth breaking the API for this, but is this an API break? This is not a keyword argument? The reason for change was to make it more clear that we are not trying to modify the incoming dataframe.

Comment (berland) [reply pyscal/utils/relperm.py:20]: (ping)

Comment (asnyv) [reply pyscal/utils/relperm.py:20]: wouldn't this break for someone that has used the `crosspoint` function in the following way?
```
some_float = crosspoint(dframe=some_df, satcol=some_col1, kr1col=some_col2, kr2col=some_col3)
``` 
Maybe it is a bad habit, but at least I like to write the argument name even though they are not required, as I find it easier to follow the code if I have an idea of what the input I give correspond to in the function I use.

File: pyscal/utils/relperm.py [hunk 2]
```diff
@@ -11,34 +11,46 @@
...
         satcol: Column name for the saturation column
         kr1col: Column name for first relperm column
         kr2col: Columnn ame for second column
```
Comment (asnyv) [inline pyscal/utils/relperm.py:23]: just noticed
```suggestion
        kr2col: Column name for second column
```

File: pyscal/utils/relperm.py (hunk 1 above)
Comment (berland) [reply pyscal/utils/relperm.py:20]: You're right, bad habit or not, this would break. Will change.

Comment (berland) [reply pyscal/utils/relperm.py]: Fixed

**Comment by asnyv** [comment]
LGTM 👍

**Review by asnyv — APPROVED** [review]

**Pull request merged**